2017.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: page
+---
+
+### Ruby for Good 2017
+
+[Join us on Slack](https://rubyforgood.herokuapp.com/) before, during, and after the event for announcements and making friends.
+
+###### Thursday June 16th
+
+* Arrive before 5:00 pm — most of the organizing team is planning on arriving at 10:00 am but feel free to beat us there!
+* 1:00 pm - 26 people on [FONZ](https://nationalzoo.si.edu/JoinFonz/join.cfm) tour of Smithsonian-Mason Institute (see below) — [reserve your spot](https://docs.google.com/spreadsheets/d/1gTi3wUj8MJLG637wdYBsasBrca53P7X-FFjXWw-SF2k/edit?usp=sharing).
+* 5:00 pm - Event kickoff and announcements
+* 5:30 pm - Team pitches, team assignments
+* 6:30 pm - Dinner, initial team discussion, setting up environments, cloning down repos, etc.
+* 8:30 pm? - Board games, computer games, socializing, and other random fun.
```

Comment:
I think it is supposed to reference 8:30 or earlier... Whenever everyone has their environments/repos ready to start the next day.